t/lib/MyModel.pm

```diff
@@ -30,7 +30,6 @@ sub testing {
     if ( $model->es_version < 1 ) {
         plan skip_all => 'Requires Elasticsearch 1.0.0';
```

Comment:
there's only one test (in `t/es/timestamp.t`) that fails on `ES 1.7.5`... I guess I can easily adapt the code to make both work.